Asterix/tests/param_file_tests.ini

```diff
@@ -26,6 +26,7 @@ onbench=True
 
     # Sampling in the detector science image
     #(lambda/Entrance_pupil_diameter in pixel
+    # For THD2 testbed : 7.42 pix / L/D for the 783.6 laser
```

Comment:
Repeating here because it is important:
Are you sure this really refers to the entrance pupil diameter? If this was measured on testbed data, the sampling of 7.42 pixels per lam/D is actually in Lyot stop diameter. We should absolutely clarify this. If this is indeed already converted to lam/D_pup, then we might at least subscript the D here to D_pup or something to make it very explicit.

Reply:
I asked the same question to Pierre yesterday and he's affirmative